### PR TITLE
Create CRUD for employee for the admin view

### PR DESCRIPTION
**Pull request by AgustinPalmaM**
Created CRUD employees for the admin view

Now we can create, read, update and delete employees

Co-authored-by: @V-Juarez <mytacam78@gmail.com>
Co-authored-by: @OwenLobato <owenlobato@gmail.com>
Co-authored-by: @ruruisu <zaddozgl@gmail.com>

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -7,11 +7,40 @@ def new
...
+    if @employee.update(employee_params)
+      redirect_to employees_path, notice: 'Employee edited ok'
+    else
+      render :edit, :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @employee = Employee.find(params[:id])
+    # @employee.destroy
```
Comment (juperameza) [inline app/controllers/employees_controller.rb:39]: Code that you don't need delete it

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -7,11 +7,40 @@ def new
...
+    if @employee.destroy
```
Comment (juperameza) [inline app/controllers/employees_controller.rb:40]: You should do a logical delete

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by V-Juarez — APPROVED** [review]

**Review by MatiasDevJam — COMMENTED** [review]
this PR has a lot of files, what do you think?

**Review by OwenLobato — APPROVED** [review]
@

**Review by OwenLobato — DISMISSED** [review]

**Review by YilKanda — CHANGES_REQUESTED** [review]
If you are going to change all the files in the check_time_clock folder to root, it is better to do it in a different PR, since only the creation of the employee's CRUD should appear in this PR.

**Review by V-Juarez — DISMISSED** [review]

**Comment by germanruzca** [comment]
Try to follow the @juperameza recommendations

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (AgustinPalmaM) [reply app/controllers/employees_controller.rb:39]: Ok it's done, thanks.

**Pull request merged**